server/cli/src/backup/README.md

```diff
@@ -0,0 +1,43 @@
+# Backup and Restore
+
+omSupply server can be backed up and restored via cli. 
+
+`NOTE` In production backups should only be done for omSupply central servers, remote sites are backed up via synchronisation
+
+### Backup
+
+To backup simply run: 
+
+**In development mode**
+
+`cargo run --bin remote service_cli -- backup`
```

Comment:
```suggestion
cargo run --bin remote_server_cli -- backup
```

I think this is what you mean, right? (And for the other ones too)

Reply:
Yeah, sorry: 63439207f4ef2463654a9fbbb3a57416c51e205f